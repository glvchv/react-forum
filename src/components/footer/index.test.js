import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '.';
import { BrowserRouter } from 'react-router-dom';

describe('Footer component', () => {
    it('should render footer', () => {
        const tree = renderer.create(
            <BrowserRouter>
                <Footer />
            </BrowserRouter>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    })
})