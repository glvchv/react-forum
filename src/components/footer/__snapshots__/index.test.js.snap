// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render footer 1`] = `
<footer
  className="wrap"
>
  <a
    className="forum"
    href="/"
    onClick={[Function]}
  >
    Forum™
  </a>
  <p>
    © 2020 - All Rights Reserved.
  </p>
</footer>
`;
